README.md

```diff
@@ -10,23 +10,37 @@ You need install:
 - [SQL server](https://www.microsoft.com/pl-pl/sql-server/sql-server-downloads)
 - [Chrome](https://www.google.com/chrome)
 - [NPM](https://nodejs.org/)
+- [dotnet-ef](https://docs.microsoft.com/en-us/ef/core/cli/dotnet)
 
 ## Usage
 
 1. Clone the repository
 
 ### Backend
 
-2. Open terminal and go to folder [`TagsterWebAPI`](./TagsterWebAPI).
-3. Build and publish application with commands below
+2. Open terminal and go to folder [TagsterWebAPI](./TagsterWebAPI).
+3. Generate sql srcipt to initialize database:
+
+- go to [TagsterWebAPI](./TagsterWebAPI/src/TagsterWebAPI) folder with .Net project
+- open [appsettings.json](./TagsterWebAPI/src/TagsterWebAPI/appsettings.json) and update `defaultConnection` for section `connectionStrings`
```

Comment:
-Add information how you can obtain connection string from your database. 
-Add information on what line is section to update